packages/docmanager/test/savehandler.spec.ts

```diff
@@ -49,6 +49,25 @@ describe('docregistry/savehandler', () => {
       });
     });
 
+    describe('#setTimer()', () => {
+      it('should reset autosave timer when disconnected', async () => {
+        jest.useFakeTimers();
+
+        jest
+          .spyOn(handler as any, '_isConnectedCallback')
+          .mockReturnValue(false);
+        jest.spyOn(handler as any, '_setTimer');
+        jest.spyOn(handler as any, '_save');
+
+        handler.saveInterval = 120;
+        handler.start();
+        jest.advanceTimersByTime(120000); // in ms
+        expect((handler as any)._setTimer).toHaveBeenCalledTimes(2);
```

Comment:
Ideally tests should interface with the public API rather than private implementation details, as the implementation may change more rapidly and the test will need modifying (or worse would get deleted). That said I do not want to hold this off so if anyone else is happy to approve as-is I will not block it.

Reply:
Good point - I've revised the test.